Dockerfile

```diff
@@ -1,77 +1,120 @@
 FROM ubuntu:focal as app
 
-# Packages installed:
-# git; Used to pull in particular requirements from github rather than pypi, 
-# and to check the sha of the code checkout.
-
-# ppa:deadsnakes/ppa; since Ubuntu doesn't ship with python 3.8 till 20, we need deadsnakes to install
-# python 3.8 on older ubuntu versions
-
-# language-pack-en locales; ubuntu locale support so that system utilities have a consistent
-# language and time zone.
-
-# python3.8-dev; to install python 3.8
-# python3-venv; installs venv module required to create virtual environments
-
-# libssl-dev; # mysqlclient wont install without this.
-
-# libmysqlclient-dev; to install header files needed to use native C implementation for 
-# MySQL-python for performance gains.
-# software-properties-common; to get apt-add-repository
-# deadsnakes PPA to install Python 3.8
-# If you add a package here please include a comment above describing what it is used for
-
-RUN apt-get update && \
-    apt-get install -y software-properties-common && \
-    apt-add-repository -y ppa:deadsnakes/ppa && \
-    apt-get update && apt-get upgrade -qy && \
-    apt-get install \
-    language-pack-en \
-    locales \
-    git \
-    libmysqlclient-dev \
-    libssl-dev \
-    build-essential \
-    python3.8-dev \
-    python3.8-distutils \
-    python3.8-venv -qy && \
-    rm -rf /var/lib/apt/lists/*
-
-ENV VIRTUAL_ENV=/edx/app/edx-notes-api/venvs/edx-notes-api
-RUN python3.8 -m venv $VIRTUAL_ENV
-ENV PATH="$VIRTUAL_ENV/bin:$PATH"
-
-
+# System requirements.
+ENV DEBIAN_FRONTEND=noninteractive
+RUN apt update && \
+  apt-get install -qy \ 
+  curl \
+  vim \
```

Comment:
I don't think we need vim here. We should remove it

Reply:
done on local